modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,4415 @@
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X GET "http://localhost:14240/gsql/v1/schema/vertices/person"
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","results":{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"UINT"},"AttributeName":"sid"},{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"id"},"Name":"person"}}
+----
+--
+====
+
+==== Parameters:
+
+NONE
+
+'''     
+
+=== create vertex usgin gsql command 
+
+`POST /gsql/v1/schema/vertices`
+
+This endpoint is used to create global vertices using gsql command
+
+==== Example:
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X POST "http://localhost:14240/gsql/v1/schema/vertices?gsql=true" -d '{"gsql":["CREATE VERTEX UserA (PRIMARY_ID user_id UINT, name STRING)", "CREATE VERTEX UserB (PRIMARY_ID user_id UINT, name STRING)"]}'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully create vertices: [UserA, UserB]"}
+----
+--
+====
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsql`
+| yes
+| this should be true, means gsql command foramt
+|===
+
+
+'''     
+
+=== create vertex (add global vertex to local graph)
+
+`POST /gsql/v1/schema/vertices`
+
+This endpoint is used to add global vertices to local graph
+
+==== Example:
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X POST -H "content-type: application/json" "http://localhost:14240/gsql/v1/schema/vertices?gsql=false&graph=empty_graph" -d '{"addVertices":["user","website"]}'
+----
+--
+Sample Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully add vertices: [user, website] on graph empty_graph"}
+----
+--
+====
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsql`
+| yes
+| this should be false, means a json format
+
+| `graph`
+| yes
+| which graph the global vertex to be added
+|===
+
+'''   
+=== create vertex (create vertex in global level)
+
+`POST /gsql/v1/schema/vertices`
+
+This api is used to create global vertex using json 
+
+==== Example:
+
+[,tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl -X POST -H "content-type: application/json" "http://localhost:14240/gsql/v1/schema/vertices?gsql=false" -d '{"createVertices":[{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"user_id"},"Name":"User5"},
+{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"user_id"},"Name":"User4"}
+]}'

```

Comment:
It's also good to put complex sample request in multiple lines, but we need to refactor it to better format